src/hooks/useSortByTopic.ts

```diff
@@ -0,0 +1,28 @@
+import { useMemo } from 'react'
+import useSafeState from './useSafeState'
+
+interface SortTopic {
+  topic: string
+  order: boolean
```

Comment:
better say  `ascendent`?

Reply:
`asc` for short. We are talking about sort, so it's pretty common and expected